### DISCUSSION TITLE
“X” is an app downloaded from the Internet

### DISCUSSION THREAD

**Discussion by DarthGigi** [Homebrew/discussions#3520] [Casks]
When I update apps via homebrew cask and then launch them macOS always gives me this prompt:
![Screenshot](https://user-images.githubusercontent.com/47110839/184132110-0956ab70-7a7c-49f2-ab67-2a54fe914be5.png)

I know that all I have to do is press "Open", however, after updating 10 apps it can get quite annoying. I was wondering if there is a way to bypass this? For example, whitelist everything downloaded from "Homebrew Cask"?

**Comment by carlocab** [comment]
You may be interested in the `--no-quarantine` flag. See `brew help install` for details.
